queries.sql

```diff
@@ -17,3 +17,32 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+-- How many animals are there? 10
+UPDATE animals SET weight_kg = weight_kg * (-1) WHERE weight_kg < 0;
+
```

Comment:
- [ ] Kindly you have missed the `How many animals are there?` query, you have instead this update query, Kindly add it.